code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -32,6 +32,23 @@
 	if(alien != IS_DIONA)
 		M.heal_organ_damage(6 * removed, 0)
 
+/datum/reagent/bicaridine/overdose(var/mob/living/carbon/M, var/alien, var/removed)
+	..()
+	var/wound_heal = 1.5 * removed
+	M.eye_blurry += (wound_heal)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		for(var/obj/item/organ/external/O in H.bad_external_organs)
+			for(var/datum/wound/W in O.wounds)
+				if(W.bleeding())
+					W.damage = max(W.damage - wound_heal, 0)
+					if(W.damage <= 0)
+						O.wounds -= W
+				if(W.internal)
```

Comment:
You could do this in one if statement because you're doing the same in both. I believe.
Can just have `if(W.bleeding() || W.internal)` or something.